src/App/Pages/Accounts/LoginPasswordlessRequestViewModel.cs

```diff
@@ -254,31 +265,66 @@ private async Task CheckLoginRequestStatus()
             }
         }
 
+        private async Task HandleLoginComplete()
```

Comment:
⛏️  Add async suffix:
```suggestion
        private async Task HandleLoginCompleteAsync()
```